extensions/wikia/Helios/HelperController.class.php

```diff
@@ -120,28 +118,7 @@ public function sendConfirmationEmail()
 			return;
 		}
 
-		$langCode = $this->getVal( 'langCode', 'en' );
-		$mailTemplate = $this->app->renderView(
-			'UserLogin',
-			'GeneralMail',
-			[
-				'language' => $langCode,
-				'type' => 'confirmation-email'
-			]
-		);
-
-		$lang = \Language::factory($langCode);
-		$mailStatus = (new GlobalStateWrapper(['wgLang' => $lang]))
-			->wrap(function() use ($user, $mailTemplate, $langCode) {
-				return $user->sendConfirmationMail(
-					false,
-					'ConfirmationMail',
-					'usersignup-confirmation-email',
-					true,
-					$mailTemplate,
-					$langCode
-				);
-			});
+		$mailStatus = $user->sendConfirmationMail( false, 'ConfirmationMail' );
 
```

Comment:
@dianafa In this case `sendConfirmationMail()` is best minimal usage with `ConfirmationMail` as default in : https://github.com/Wikia/app/blob/dev/includes/User.php#L3869 , other params don't matter as execution of https://github.com/Wikia/app/blob/dev/includes/User.php#L3909 depends only on passed `$mailtype` and is returning before any other param would be used.